merge.go

```diff
@@ -0,0 +1,100 @@
+package neo4j
+
+import (
+	"fmt"
+
+	"github.com/cloudprivacylabs/lpg"
+	"github.com/cloudprivacylabs/lsa/pkg/ls"
+	"github.com/neo4j/neo4j-go-driver/v4/neo4j"
+)
+
+func merge(g1, g2 *lpg.Graph) (*lpg.Graph, error) {
```

Comment:
Name these as memGraph, dbGraph, also get a map[*Node]int for node-id mapping

Return new db graph (dbGraph), also return the slice of actions to perform on the db.